app/controllers/listings_controller.rb

```diff
@@ -312,12 +312,14 @@ def create
         ).html_safe
 
         # Onboarding wizard step recording
-        state_changed = Admin::OnboardingWizard.new(@current_community.id)
-          .update_from_event(:listing_created, @listing)
-        if state_changed
-          report_to_gtm({event: "km_record", km_event: "Onboarding listing created"})
+        if @current_user.has_admin_rights?
```

Comment:
`has_admin_rights?` returns `true` for both marketplace admins AND super admins. Is that on purpose or should be include only marketplace admins and ignore super admins?